Oswaldo Vazquez Aparicio/30 Agosto/hackerrank/JavaMap.java

```diff
@@ -1,19 +1,20 @@
 package hackerrank;
 
 import java.util.HashMap;
+import java.util.Map;
 import java.util.Scanner;
 
 public class JavaMap {
 
     private static final Scanner scanner = new Scanner(System.in);
-    private static HashMap<String, Integer> phoneBook;
+    private static Map<String, Integer> phoneBook = new HashMap<>();
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableListList, userSet, etc...